require.js

```diff
@@ -0,0 +1,4 @@
+const $ = window.jQuery = require('jquery')
+require('../node_modules/semantic-ui/dist/semantic.min.js')
```

Comment:
`<script src="/node_modules/semantic-ui/dist/semantic.min.js"></script>` is better than this line.